specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11533,7 +11533,7 @@
         "commandLine": {
           "type": "string",
           "title": "The command line of the Job Manager task.",
-          "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
+          "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux. The command line had better use relativate path of task execution directory, or use the Batch provided environment variables(https://docs.microsoft.com/en-us/azure/batch/batch-compute-node-environment-variables)."
```

Comment:
If the command line refers to file paths, it should use a relative path (relative to the task working directory), or use the Batch provided environment variables (https://docs.microsoft.com/en-us/azure/batch/batch-compute-node-environment-variables)

Reply:
This applies to all copies of this description, since there are a few